addons/hr_holidays/models/hr_leave_type.py

```diff
@@ -52,10 +53,57 @@ class HolidaysType(models.Model):
     virtual_remaining_leaves = fields.Float(compute='_compute_leaves', string='Virtual Remaining Leaves',
         help='Maximum Leaves Allowed - Leaves Already Taken - Leaves Waiting Approval')
 
-    double_validation = fields.Boolean(string='Apply Double Validation',
+    double_validation = fields.Boolean(string='Apply Double Validation', compute='_compute_double_validation',
```

Comment:
I agree that we should find a way to avoid duplication, but this has to be displayed in the view (maybe we can just keep it as a computed field and use it only for interface)

Reply:
Can't we just display validation_type ? Is it really required ?

Reply:
Ok I'm displaying validation_type instead of double_validation